app/models/project.rb

```diff
@@ -9,12 +12,19 @@ class Project < ApplicationRecord
 
   class << self
     def maya_top_level_dir
-      dir = "#{Dir.home}/maya/projects/."
```

Comment:
Is there some issue here? IIRC I added the ending `.` for the file picker's sake because it'll start at the parent (which I want to be `projects`, not `maya`.